Source/WebCore/rendering/svg/legacy/LegacyRenderSVGRoot.cpp

```diff
@@ -362,7 +362,12 @@ LayoutRect LegacyRenderSVGRoot::clippedOverflowRect(const RenderLayerModelObject
     if (isInsideEntirelyHiddenLayer())
         return { };
 
-    FloatRect contentRepaintRect = m_localToBorderBoxTransform.mapRect(repaintRectInLocalCoordinates());
+    FloatRect repaintRectInLocalCoordinates;
+    if (context.options.contains(VisibleRectContextOption::DoNotUseSVGFastRepaintRectInLocalCoordinates))
+        repaintRectInLocalCoordinates = this->repaintRectInLocalCoordinates();
+    else
+        repaintRectInLocalCoordinates = this->fastRepaintRectInLocalCoordinates();
```

Comment:
I think you are changing all the calls to `repaintRectInLocalCoordinates()` to `fastRepaintRectInLocalCoordinates()` unconditionally in all places expect this one, the one in `SVGRenderSupport::clippedOverflowRectForRepaint()` and the ones in `checkIntersection()` and `checkEnclosure()`

So I think it is worth looking at adding a calculation method flag to `repaintRectInLocalCoordinates()` and default it to `Fast` and removing `fastRepaintRectInLocalCoordinates()`.

Reply:
Done.